sdk/src/sharedTest/java/io/customer/sdk/data/store/DeviceStoreTest.kt

```diff
@@ -36,22 +32,22 @@ internal class DeviceStoreTest {
     }
 
     @Test
-    fun `verify build attributes in device`() {
-        deviceStore.deviceBrand `should be equal to` "Google"
-        deviceStore.deviceModel `should be equal to` "Pixel 6"
-        deviceStore.deviceManufacturer `should be equal to` "Google"
-        deviceStore.deviceOSVersion `should be equal to` 30
+    fun verifyBuildAttributesInDevice() {
+        deviceStore.deviceBrand shouldBeEqualTo "Google"
```

Comment:
Out of curiosity, why were these changes made? Was the previous version not that reader friendly?

Reply:
I encountered [this issue](https://github.com/MarkusAmshove/Kluent/issues/53) which required me to change our dependency from using `kluent` to `kluent-android`. Running tests on the Android OS via instrumentation testing throws a compile time error saying that the old syntax couldn't compile. 

It's the same compile time error if you create a test function like:
```kt
    @Test
    fun `given X, expect Y`() {}   
```

compared to:
```kt
    @Test
    fun givenX_expectY() {}   
```